baby-bear/src/baby_bear.rs

```diff
@@ -18,6 +18,12 @@ pub struct BabyBear {
     value: u32,
 }
 
+impl BabyBear {
+    pub(crate) const fn new(n: u32) -> Self {
```

Comment:
Could we use `from_canonical_u32`? I know it's a bit verbose but I think it's good to make it clear that the argument is a canonical int, not Monty encoded int

Reply:
I tried this before, but the case is we need to init const field element, which requires a const constructor. `from_canonical_u32` is defined by a trait, so we cannot directly use it in a const constructor. I think the only difference is an assertion. If you think that is necessary, we can add it. or maybe we can add some comments to this method?

Reply:
I added a short comment for this method. If you have any other elegant option, please let me know.